packages/@uppy/webcam/src/index.js

```diff
@@ -54,6 +54,10 @@ module.exports = class Webcam extends UIPlugin {
   // eslint-disable-next-line global-require
   static VERSION = require('../package.json').version
 
+  // enableMirror is used to toggle mirroring, for instance when discarding the video,
+  // while `opts.mirror` is used to remember the initial user setting
+  #enableMirror = this.opts?.mirror ?? true
```

Comment:
`this.opts` is always `undefined` at this point (class field assignments are executed before class constructor):

```js
new class {
  constructor() { console.log('second') }
  #priv = console.log('first'); 
} // logs "first", then "second"
```
The assignment needs to stay in the constructor body after `this.opts` assignment.
```suggestion
  #enableMirror
```

Reply:
I expected the same, but I tested it locally and the current code works. Nonetheless I updated the code.